content/ko/docs/concepts/workloads/pods/pod.md

```diff
@@ -0,0 +1,151 @@
+---
+reviewers:
+title: Pods
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+_파드_ 는 쿠버네티스에서 생성되고 관리될 수 있는 배포 가능한 최소 컴퓨팅 단위이다.
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## 파드는 무엇인가?
+(여러개 또는 단일의) _파드_ 는 도커 컨테이너 처럼 스토리지/네트워크를 공유하는 하나 이상의 컨테이너 그룹이다.
+파드의 컨텐츠들은 항상 함께 배치되고 같은 일정에 따라 실행된다. 파드 모델은 "논리 호스트" 라는 특별한 어플리케이션이다. 이것은 하나 이상으로 강하게 연관되어 있는 어플리케이션 컨테이너이다. 같은 물리적 또는 가상의 머신에서 실행되는 것은 같은 논리적 호스트에서 실행되고 있는 것을 의미한다.
+
+Kubernetes는 Docker 이외에도 많은 컨테이너 런타임을 지원하지만 Docker는 가장 일반적으로 알려진 런타임이며 Docker 용어로 파드를 설명하는 데 도움이된다.
+
+포드의 공유 컨텍스트는 Linux 네임 스페이스, cgroup 및 도커 컨테이너를 격리하는 것과 같이 잠재적으로 다른 격리 요소들이다. 포드의 컨텍스트 내에서 개별 응용 프로그램은 추가적으로 하위 격리가 적용된다.
+
+컨테이너들 안의 파드는 IP주소와 포트를 공유하고, 서로를 `localhost` 를 통해 찾을 수 있다. 그들은 또한 SystemV 세마포어나, POSIX 공유 메모리와 같이 표준 프로세스 간 통신을 사용하는 것들과 각각 의사 소통을 할 수 있다.
+다른 파드의 컨테이너에는 고유 한 IP 주소가 있고, [특별한 구성](/docs/concepts/policy/pod-security-policy/) 없이는 IPC에 의해서 통신 할 수 없다. 그리고 컨테이너는 주로 서로의 IP 주소를 통해 소통한다.
+
+또한 파드 안의 어플리케이션은 파드의 일부로 정의되, 각각의 어플리케이션의 파일시스템에 마운트가 할 수 있도록 만들어진 공유 볼륨에 엑세스 할 수 있다.
+
+[도커](https://www.docker.com/)의 구조 관점에서 보면 파드는 공유 네임스페이스와 공유 [볼륨](/docs/concepts/storage/volumes/)을 가진 도커 컨테이너 그룹으로 모델링 된다.
+
+개별 어플리케이션 컨테이너와 같이, 파드는 상대적으로 수명이 짧은 엔터티로 간주된다. [파드의 생애](/docs/concepts/workloads/pods/pod-lifecycle/)에서 논의된 것 과 같이, 파드가 만들어지고 고유한 ID (UID)가 할당되고, 재시작 정책에 따라서 종료 또는 삭제될 때 까지 노드에 예약된다.  
+노드가 종료되면 해당 노드로 예약 된 파드는 제한시간이 지나면 삭제되도록 예약된다. UID로 정의된 파드는 새로운 노드에 "재예약" 되지 않는 대신 같은 이름이지만, 새로운 UID를 갖는 동일한 파드로 교체될 수 있다.
+더 자세한 내용은 [레플리케이션 컨트롤러](/docs/concepts/workloads/controllers/replicationcontroller/)를 참조한다.
+
+볼륨과 같이 포드와 동일한 수명이 있다고 하면, UID를 포함한 해당 파드가 존재하는 한 그것도 존재한다는 것을 의미한다.
+어떤 이유로든 해당 파드가 삭제 된 경우 동일한 대체품이 만들어 지더라도 관련된 것(예 : 볼륨) 또한 삭제되고 새로 만들어진다.
+
+{{< figure src="/images/docs/pod.svg" title="pod diagram" width="50%" >}}
```

Comment:
Should we translate "pod diagram"?

Reply:
@sftim I think it just html code. so you don't need translate.

Reply:
“pod diagram” will be visible to at least some readers via, eg, a tool tip.
It should be translated into Korean.

Reply:
@sftim I understand! okay I will translate to that. thx!